app/addons/documents/routes-documents.js

```diff
@@ -154,6 +163,10 @@ function (app, FauxtonAPI, BaseRoute, Documents, Changes, ChangesActions, Databa
         this.rightHeader.showQueryOptions();
       },
 
+      replicatorShowDocsOnDefault: function () {
```

Comment:
Maybe instead of having a set url for the replicator. Rather have a function in the all_docs thats checks if the databaseName matches an item in an array. If so turn include_docs on.

``` js
if ( ['_replicator', '_user'].indexOf(databaseName) > -1) {
 docParams.include_docs = true;
}


```